examples/Reranking_search_results_with_cross-encoders.ipynb

```diff
@@ -0,0 +1,805 @@
+       "9        2      2  how do bi-encoders work for sentence embeddings   \n",
+       "\n",
+       "                                            document prediction  logprobs  \\\n",
+       "0  Vec2Sent: Probing Sentence Embeddings with Nat...        Yes -0.000157   \n",
+       "1  SBERT studies Meaning Representations: Decompo...        Yes -0.000248   \n",
+       "2  In Search for Linear Relations in Sentence Emb...        Yes -0.000260   \n",
+       "3  Sentence-T5: Scalable Sentence Encoders from P...        Yes -0.000269   \n",
+       "4  SentPWNet: A Unified Sentence Pair Weighting N...        Yes -0.000303   \n",
+       "5  Relational Sentence Embedding for Flexible Sem...        Yes -0.001242   \n",
+       "6  Efficient and Flexible Topic Modeling using Pr...        Yes -0.139526   \n",
+       "7  Learning Probabilistic Sentence Representation...        Yes -0.252695   \n",
+       "8  Non-Linguistic Supervision for Contrastive Lea...        Yes -0.319398   \n",
+       "9  Semantic Composition in Visually Grounded Lang...        Yes -0.474471   \n",
+       "\n",
+       "   probability  cleaned_probability  \n",
+       "0     0.999843             0.999843  \n",
+       "1     0.999752             0.999752  \n",
+       "2     0.999740             0.999740  \n",
+       "3     0.999731             0.999731  \n",
+       "4     0.999697             0.999697  \n",
+       "5     0.998759             0.998759  \n",
+       "6     0.869771             0.869771  \n",
+       "7     0.776705             0.776705  \n",
+       "8     0.726587             0.726587  \n",
+       "9     0.622214             0.622214  "
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Return reranked results\n",
+    "reranked_df = output_df.sort_values(by=[\"cleaned_probability\"], ascending=False).reset_index()\n",
+    "reranked_df.head(10)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "610b2c7f",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'Vec2Sent: Probing Sentence Embeddings with Natural Language Generation: We introspect black-box sentence embeddings by conditionally generating from\\nthem with the objective to retrieve the underlying discrete sentence. We\\nperceive of this as a new unsupervised probing task and show that it correlates\\nwell with downstream task performance. We also illustrate how the language\\ngenerated from different encoders differs. We apply our approach to generate\\nsentence analogies from sentence embeddings.'"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "reranked_df['document'][0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f372d311",
+   "metadata": {},
+   "source": [
+    "## Consolidation\n",
```

Comment:
What does Consolidation mean?

Was expecting this to say Conclusion :)

Reply:
In any case, there might be some room to add explanation of how this compares and why it might be beneficial to use. Right now it just kind of abruptly ends without any discussion of how much better these results are than an alternative.

Reply:
Right, I sometimes say Consolidation but it is harder to understand, updated to Conclusion and gave a few suggestions of use cases and next steps.